packages/editor/src/client/routes/comments/commentEditView.tsx

```diff
@@ -86,196 +138,228 @@ const CommentEditView = memo(() => {
     selectedTags
   ])
 
-  /**
-   * Helper function to parse comment revision input object out of full revision fragment.
-   * @param comment
-   */
-  function getLastRevision(comment: FullCommentFragment): CommentRevisionUpdateInput | undefined {
-    const revisions = comment.revisions
-    if (!revisions.length) {
-      return
-    }
-    const lastRevision = revisions[revisions.length - 1]
-    const parsedRevision = {
-      title: lastRevision?.title,
-      lead: lastRevision?.lead,
-      text: lastRevision?.text
-    } as CommentRevisionUpdateInput
-    return parsedRevision
-  }
+  const ratingOverrides = useMemo(
+    () =>
+      ratingSystem?.ratingSystem.answers.map(answer => ({
+        answerId: answer.id,
+        name: answer.answer,
+        value:
+          comment?.overriddenRatings?.find(override => override.answerId === answer.id)?.value ??
+          null
+      })) ?? [],
+    [comment, ratingSystem]
+  )
 
-  /**
-   * Check, if revision object differs from original. Used to decide, whether to create a new revision.
-   */
-  function revisionChanged(): boolean {
-    if (!comment) {
-      return true
-    }
-    const originalVersion = getLastRevision(comment)
-    if (JSON.stringify(originalVersion) !== JSON.stringify(revision)) {
-      return true
-    }
-    return false
-  }
+  const ratingOverridePossibleValues = [
+    {label: t('commentEditView.noOverride'), value: null},
+    {label: '1', value: 1},
+    {label: '2', value: 2},
+    {label: '3', value: 3},
+    {label: '4', value: 4},
+    {label: '5', value: 5}
+  ]
 
   async function updateComment() {
     if (!comment) {
       return
     }
+
     await updateCommentMutation({
       variables: {
         id: comment.id,
-        revision: revisionChanged() ? revision : undefined,
+        revision: hasRevisionChanged(comment, revision) ? revision : undefined,
```

Comment:
If you create a new comment, add a title & lead and override the comment rating and want to save, it fails.

`Variable "$ratingOverrides" got invalid value { __typename: "overriddenRating", answerId: "cl9gz0eao13080010aoan2iezw", value: 5 } at "ratingOverrides[0]"; Field "__typename" is not defined by type CommentRatingOverrideUpdateInput.`

Reply:
Can you still reproduce this? I can't.

Reply:
Ahh, saving then reloading then overriding will produce this error